src/stories/controls/AlertControl.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from "vitest";
+import { screen } from "@testing-library/react";
+import { render } from "../../common/test-render";
+import { JSONSchema } from "json-schema-to-ts";
+
+
+const schema = {
+  type: "object",
+  properties: { 
+    text: { type: "string" }, 
+    options: {
+      type: "object", 
+      properties: {
+        type: {
+          type: "string"
+        }
+      }
+    }
+  }
+} satisfies JSONSchema;
+
+test("defaults to info", async () => {
+  render({
+    schema: schema,
+    uischema: {
+        type: "VerticalLayout",
+        elements: [
+          {
+            type: "Label",
+            text: "An African swallow, maybe -- but not a European swallow, that's my point.",
+            options: {}
+          }
+        ],
+      },
+  });
+  const input = await screen.findByRole("alert");
+  expect(input).toHaveClass("ant-alert-info");
```

Comment:
you can omit this expect.
- we don't control antd classnames and they don't show up in our code, so i don't like writing tests based on those classnames
- `findBy*` will reject (and the test will fail) if it doesn't find the alert by its aria role, so it is sufficient by itself

Reply:
The intent of this test was to assert that the default mode whe not passed `options` is "info". What you propose it more of a "does it render" test then right?